### PR TITLE
[INJICERT-14] add some field notes from running it on Mac & Windows

### PR DESCRIPTION
**Pull request by vharsh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,15 @@ The following steps will help you to setup Sunbird RC and Esignet services using
 * Docker Compose (2.25)
 
 ## Installation
-### Steps to setup Insurance credential use case  
+
+### Steps to setup Insurance credential use case
+
+NOTE:
+
+- Apple Silicon Mac users should export or set `DOCKER_DEFAULT_PLATFORM=linux/amd64` before running the `install.sh` and use GNU `sed` to run the script over BSD `sed`. A simple way to do it would be to replace all instances of `sed` in the script with `gsed`. The former change is required to bring-up Vault cleanly without any unsealing errors and the latter had to be done because `sed` scripts are usually not portable across platforms.
```
Comment (challabeehyv) [inline README.md:21]: can you add this to a troubleshoot section in the bottom along with the error

File: README.md [hunk 2]
```diff
@@ -29,24 +31,28 @@ Execute installation script
...
    * [Registry](https://github.com/Sunbird-RC/sunbird-rc-core)
 5. Post Sunbird installation, proceed to create an issuer and credential schema. Refer to the API schemas available [here](https://github.com/Sunbird-RC/sunbird-rc-core/tree/main/api-documentation).
-6. Add the jar file of Digital Credential Stack(DCS) plugin implementation in [loader_path](docker-compose-esignet/loader_path) the GitHub link for the repository can be found [here](https://github.com/mosip/digital-credential-plugins).
+    * Set the hostname of the endpoints correctly as per your docker setup
+    * Now generate a DID, create a credential schema and create an issuance registry
+        * take note of $.schema[0].author & $.schema[0].id from the create credential schema request
+6. Add the jar file of Digital Credential Stack(DCS) plugin implementation in [loader_path](docker-compose-esignet/loader_path) the GitHub link for the repository can be found [here](https://github.com/mosip/digital-credential-plugins) if you wish to build it locally or you can download from [here](https://mvnrepository.com/artifact/io.mosip.esignet.sunbirdrc/sunbird-rc-esignet-integration-impl).
```
Comment (challabeehyv) [inline README.md:48]: links are interchanged

Comment (challabeehyv) [reply README.md:48]: can we add the steps for vc download as well? @vharsh

File: docker-compose-sunbird/docker-compose.yml [hunk 3]
```diff
@@ -163,7 +163,7 @@ services:
     volumes:
       - ./imports/nginx/nginx.conf:/etc/nginx/conf.d/default.conf
     ports:
-      - '80:80'
```
Comment (challabeehyv) [inline docker-compose-sunbird/docker-compose.yml:166]: can we keep this 80 inside the container and 8000 to be exposed outside

File: docker-compose-sunbird/imports/nginx/nginx.conf [hunk 4]
```diff
@@ -1,5 +1,5 @@
 server {
-    listen 80;
+    listen 8000;
```
Comment (challabeehyv) [inline docker-compose-sunbird/imports/nginx/nginx.conf:2]: lets keep 80 as its standard practice

File: README.md (hunk 2 above)
Comment (vharsh) [reply README.md:48]: I've reworked the above sentence to make it less confusing.

File: docker-compose-sunbird/imports/nginx/nginx.conf (hunk 4 above)
Comment (vharsh) [reply docker-compose-sunbird/imports/nginx/nginx.conf:2]: @vishwa-vyom had asked this to be updated to a non-privileged port earlier in an internal discussion, should I revert this change?

File: docker-compose-sunbird/docker-compose.yml (hunk 3 above)
Comment (vharsh) [reply docker-compose-sunbird/docker-compose.yml:166]: makes sense, I'll make this change

File: docker-compose-sunbird/imports/nginx/nginx.conf (hunk 4 above)
Comment (challabeehyv) [reply docker-compose-sunbird/imports/nginx/nginx.conf]: this will be used to route internally so let this be 80

**Review by challabeehyv — APPROVED** [review]

**Review by challabeehyv — APPROVED** [review]

File: docker-compose-sunbird/docker-compose.yml (hunk 3 above)
Comment (challabeehyv) [reply docker-compose-sunbird/docker-compose.yml:166]: ok

**Review by challabeehyv — APPROVED** [review]

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**